### PR TITLE
Re-order rules of "base"

### PR DESCRIPTION
**Pull request by akai**
Based on http://eslint.org/docs/rules/

### CODE REVIEW

**Comment by ljharb** [comment]
I'm marking this as a breaking change just in case someone is relying on direct requires of the rule files.

File: packages/eslint-config-airbnb-base/rules/style.js [hunk 1]
```diff
@@ -125,6 +144,19 @@ module.exports = {
     // disallow use of the Object constructor
     'no-new-object': 2,
 
+    // disallow use of unary operators, ++ and --
+    'no-plusplus': 0
```
Comment (ljharb) [inline packages/eslint-config-airbnb-base/rules/style.js]: this is missing a comma after it

**Comment by ljharb** [comment]
This LGTM - before I merge it, I want to cut a release of "base", and then I'll want this rebased, and then this can be merged :-)

**Comment by akai** [comment]
Maybe we should migrate import's rules/settings to a separate config

**Comment by ljharb** [comment]
That's probably a good idea actually, even though `import` is ES6. Let's do that in this PR - `rules/imports` or something?

File: packages/eslint-config-airbnb-base/rules/es6.js [hunk 2]
```diff
@@ -10,9 +10,6 @@ module.exports = {
       'objectLiteralDuplicateProperties': false
```
Comment (akai) [inline packages/eslint-config-airbnb-base/rules/es6.js]: I can't find these options [here](http://eslint.org/docs/user-guide/configuring#specifying-parser-options). Is it useful?

Comment (ljharb) [inline packages/eslint-config-airbnb-base/rules/es6.js]: It's documented [here](http://eslint.org/docs/user-guide/migrating-to-2.0.0#language-options). Still useful :-)

**Pull request merged**